### DISCUSSION TITLE
Media Player Subtitles Support

### DISCUSSION THREAD

**Discussion by Sesshoumaru-sama** [filebrowser/filebrowser#1641] [Ideas]
I have a large collection of anime videos. I noticed that the player of file-browser can playback the media but does not seem to support subtitles (or switching of subs and audio). Is that a known restriction or something I have to configure?

**Comment by ramiresviana** [comment]
FileBrowser does support subtitles with the `.vtt` format.

**Reply by Sesshoumaru-sama** [reply]
SRT and ASS would be enough.
